MultiPage-Clone/css/resume.css

```diff
@@ -0,0 +1,647 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Quicksand;
+  src: url(/MultiPage-Clone/fonts/Quicksand/Quicksand-VariableFont_wght.ttf);
+  font-family: poppins;
+  src: url(MultiPage-Clone/fonts/poppins);
+}
+
+/* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --clr-Dark-beige: #e6dacd;
+  --clr-Light-beige: #f4ece6;
+  --clr-Royal-blue: #0150fd;
+  --clr-Black: #000;
+  --clr-White: #fff;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 150px;
+  background: var(--clr-White);
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 999;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  margin-left: 2rem;
+}
+
+.logo1 {
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo1 img {
+  width: 1rem;
+  height: 1rem;
+}
+
+.logo1 span:nth-child(3) {
+  margin-left: 5px;
+  font-size: 20px;
+}
+
+.logo1 :nth-child(2) {
+  font-family: poppins;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.logo :nth-child(4) {
+  font-size: 20px;
+}
+
+.logo p {
+  font-family: Quicksand;
+  font-size: 16px;
+}
+
+.nav-links {
+  list-style: none;
+  display: flex;
+  text-transform: uppercase;
+}
+
+ul {
+  margin-right: 2rem;
+}
+
+.nav-links li {
+  padding: 0 20px;
+}
+
+.nav-links a {
+  color: var(--clr-Black);
+  text-decoration: none;
+  font-size: 15px;
+  font-family: Quicksand;
+}
+
+.nav-links li:first-child a {
```

Comment:
For resume it should be the second child.
But usually people don't count children in css. They just use an extra class. For example "active" or something like this.